src/app/pages/subjects/category-section/book/book.js

```diff
@@ -0,0 +1,53 @@
+import BaseView from '~/helpers/backbone/view';
+import router from '~/router';
+import {on, props} from '~/helpers/backbone/decorators';
+import {template} from './book.hbs';
+import GetThisTitle from '~/components/get-this-title/get-this-title';
+
+@props({
+    template: template,
+    regions: {
+        getThis: '.get-this-title-container'
+    }
+})
+export default class Book extends BaseView {
+
+    @on('click img')
+    selectMe() {
+        this.model.set('selectedBook', this.isSelected() ? false : this.data);
+    }
+
+    @on('click .cta > .btn')
+    goToDetails() {
+        router.navigate(`/details?${ this.data.title}`, {
+            trigger: true
+        });
+    }
+
+    constructor(data, model) {
+        super();
+
+        this.templateHelpers = {
+            coverUrl: data.coverUrl
+        };
+        this.data = data;
+        this.model = model;
+
+        this.model.on('change:selectedBook', () => this.setState());
+    }
+
+    setState() {
+        this.el.classList.toggle('selected', this.model.get('selectedBook') === this.data);
+    }
+
+    onRender() {
+        this.el.classList.add('cover');
```

Comment:
Maybe this should be added to the BaseView so it can be added as a property.  Currently all views that are injected are wrapped in a `div` since that's the easiest way to manage/append them.

I had been considering trying to create them using a documentFragment instead so that extra div wasn't created (which would break this); however, since every template should currently be wrapped in a namespace (since scoped styling and the Shadow DOM aren't fully supported yet), I could see this being cleaner than adding the namespaced div into the template itself.

Unfortunately, until scoped styling is widely supported, I think this may break on browsers that DO support scoped styling.

Using this convention, the view is being created with the following DOM structure:
```html
<div class="cover">
    <div>
        // content (including @import with scoped styles)
    </div>
</div>
```
Since scoped styling only applies to all content inside the parent element, which would be the inner `<div>`, there'd be no way to style the outer div with the `cover` class applied to it.

Therefore, I think for the time being it is better to wrap the template in `<div class="cover">` as is done on other pages.